README.md

```diff
@@ -48,57 +48,81 @@ We love and welcome contributions to our front-end repository which can be found
 npm install
 ```
 
-## 2. Set up Husky Hooks 🐕️
+<!-- ## 2. Set up Husky Hooks 🐕️
+
+/*This isn't implemented as a script anywhere */
 
 This will automatically run linting before commits and reduce common contributor mistakes.
 
 ```sh
 npm run prepare
-```
+``` -->
 
-## 3. Setting up the Database 🐘
+## 2. Setting up the Database 🐘
 
 For simplicity, we use firebase!
 
 For security, we run a local database so we don't break production!
 
 (optional) You can point to different database environments simply by changing the firebase url and variables in the `.env.dev` file.
 
-### 3.a Setup Firebase CLI
+### 2.a Setup Firebase CLI
 
 #### a.1 Install the Firebase CLI if you do not have it installed on your machine:
 
 ```sh
 npm -g i firebase-tools
 ```
 
-#### a.2 Install Java
+#### a.2 Install Java to the path
 
 on a mac, we recommend using [homebrew](https://brew.sh/)
 
 ```sh
-brew install java
+brew install openjdk
 ```
 
 on a windows pc, we recommend using [chocolatey](https://chocolatey.org/)
 
 ```sh
-choco install java
+choco install openjdk
 ```
 
-#### a.3 Log into Firebase CLI:
+you can check on both operating systems if it installed properly by running the following command in a new terminal
 
 ```sh
-firebase login
+java -version
 ```
 
-#### a.4 Set the CLI to use the project:
+if installed properly it will display a message similar to this
 
 ```sh
-firebase use --add
+java version "1.8.0_411"
 ```
 
-Select "Use an existing project"
+### a.3 Upgrade to most recent JDK
+
+If the version is below 11 you will need to visit [Oracle](https://www.oracle.com/jp/java/technologies/downloads/) and download the latest version based on your OS
+
+Once updated run this command again
+
+```sh
+java -version
+```
+
+You should see an updated version of Java like below
+
+```sh
+java version "22.0.1" 2024-04-16
+```
+
+#### You might have to restart vs code for step a.2 and a.3 all of these to take effect in the project
+
+#### a.3 Log into Firebase CLI:
+
```

Comment:
The `firebase use --add` need to be in the readme in case people have other projects running with firebase.(this was the case for me), so you need to be able to point to the correct project.

https://firebase.google.com/docs/cli

Reply:
Then this needs to be given conditionals if you have other projects. Technically I do not belong to the firebase for findadoc. So this is what I saw when I used that command 

```
$ firebase use --add
(node:6176) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)? Which project do you want to add? (Use arrow keys)
```

And if I use the arrow keys I get this.

`? Which project do you want to add?`

There are no options. So I could try this forever and keep getting this error
`Error: An unexpected error has occurred.`

So we should explain if you are part of the firebase you can select it otherwise you can skip this step? What do you think?

Reply:
Fixed